appcues/src/main/java/com/appcues/trait/appcues/effects/ConfettiEffects.kt

```diff
@@ -0,0 +1,47 @@
+package com.appcues.trait.appcues.effects
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import com.appcues.data.mapper.styling.mapToColors
+import com.appcues.data.model.styling.ComponentStyle
+import nl.dionsegijn.konfetti.compose.KonfettiView
+import nl.dionsegijn.konfetti.core.Party
+import nl.dionsegijn.konfetti.core.Position.Relative
+import nl.dionsegijn.konfetti.core.emitter.Emitter
+import nl.dionsegijn.konfetti.core.models.Size
+import java.util.concurrent.TimeUnit.MILLISECONDS
+
+private const val EMITTER_INTENSITY_RAW = 200
+
+@Composable
+internal fun ConfettiEffect(style: ComponentStyle?, duration: Int, intensity: Double) {
+    // default colors in case colors style property is empty
+    val colors = remember {
+        val styleColors = style?.colors ?: listOf()
+
+        styleColors.ifEmpty { listOf("#D62828", "#FCBF49", "#003049").mapToColors() }.map { it.toInt() }
```

Comment:
should match ios? https://github.com/appcues/appcues-ios-sdk/pull/552/files#diff-c8cb90b8ae7773fe68f9dd49a35830b84bb5f12f0cdc709ef07f7baa65a69d60R129